src/check-pr.ts

```
@@ -45,16 +45,27 @@ export async function hasBeenApproved(): Promise<void> {
 async function isApprover(): Promise<boolean> {
   await moveSignOffFile()
 
+  type Approver = {
+    name: string
+    team: string
+  }
+
   const data = readFileSync(`${input.name}.json`, 'utf-8')
   const approvers = JSON.parse(data)
+  const list: [Approver] = approvers.list
```

I would consider adding a test for this property so you can gracefully error out if its not present.